docs/docs/09-tooling-integration.md

```diff
@@ -3,78 +3,11 @@ id: tooling-integration
 title: Tooling Integration
 permalink: tooling-integration.html
 prev: more-about-refs.html
-next: addons.html
+next: language-tooling.html
 ---
 
-Every project uses a different system for building and deploying JavaScript. We've tried to make React as environment-agnostic as possible.
+We've tried to make React as environment-agnostic as possible.  People use React in a variety of languages (JavaScript, TypeScript, ClojureScript, etc) and in a variety of environments (web, iOS, Android, NodeJS, Nashorn, etc).  There are many tools to help you build great applications, and this section provides information that will help you use these related tools to help build your application.
 
-## React
+* [Language Tooling](/react/docs/language-tooling.html) - provides information on using transpilers (most notably Babel to transform JSX) as configured for use with React.
```

Comment:
I want to leave it open to other traspilers/languages.  How about "describes how to set up tools like Babel to transpile JSX for a better development experience."